README.md

```diff
@@ -4,111 +4,209 @@ An API for validating survey schemas.
 make test-unit
 ```
 
-To run the app's unit tests and ajv validator tests:
+Make sure you don't already have Ajv running on localhost:5002 by running `lsof -i tcp:5002` if you do make a note of the PID and then run `kill -9 PID`, replacing "PID" with the process id from the previous command.
 
+Run the Ajv validator tests:
+``` shell
+make test-ajv
 ```
+
+To run the app's unit tests and Ajv validator tests:
+``` shell
 make test
 ```
 
-To test the apps functionality:
-```
+#### Test the local validator app against runner schemas
+
+Spin validator up with:
+``` shell
 make run
 ```
 
-Then, in another terminal window/tab, navigate to a checked out copy of eq-survey-runner:
+Then, in another terminal, navigate to a checked out copy of `https://github.com/ONSdigital/eq-questionnaire-runner` and run:
+``` shell
+make validate-test-schemas
 ```
-make test
+This will run the validator against all runner test schemas.
+
+Or you can run it against a specific runner schema, to do this:
+- set the runner vars:
+    - `SCHEMA_PATH` to the path of the schema file (defaults to `./schemas/test/en/`)
+    - `SCHEMA` to the schema file name without the `.json`
+- then run:
+``` shell
+make validate-test-schema
 ```
 
-# Installing node dependencies
+## Formatting/linting json
 
-In the eq-schema-validator directory, install node version manager (nvm) and node using the following commands:
+Run the following to format all json files in the schemas directory:
 
-```
-brew install nvm
-nvm install
+``` shell
+make format
 ```
 
-Install node dependencies:
+Run the following to lint all schemas, test schemas and Ajv files in the repo:
 
-```
-npm install
+``` shell
+make lint
 ```
 
-To run the ajv validator tests:
+## Running with Docker
 
-```
-make test-ajv
+To install Docker run:
+
+``` shell
+brew install docker
 ```
 
-## Formatting json
+On MacOS install container runtimes, eg. Colima:
 
-Run the following to format all json files in the schemas directory:
+``` shell
+brew install colima
+```
 
+Make sure Colima is started every time you want to use Docker images:
+
+``` shell
+colima start
 ```
-make format
-````
 
-## Validating with ajv
+When PRs are merged in this repo there is a GitHub workflow that builds 2 Docker images one for validator and one for the AJV Validator and then pushes them to our GAR in GCP. These images can then be pulled down and run locally with Docker. These images are pulled down and run from Runner when `make run validator` is run which uses the `docker-compose-schema-validator.yml` script.
 
-Also included is a node based version of the json schema validation which may be used during development to assist with
-debugging errors. This returns more errors than we'd currently like due to the way polymorphism works for each of our
-blocks.
+You can do this using these commands:
 
-Run the ajv (server) based version of validator.
+- Validator:
 
+``` shell
+docker run -it -p 5001:5001 onsdigital/eq-questionnaire-validator:latest
```

Comment:
I think we're using GAR now, so this Docker Hub image paths would be outdated?

Reply:
As discussed - will probably need to build locally and run it instead 👍

Reply:
Waiting for confirmation if we're happy with exposing full GAR paths in a public repo (probably okay, but need to get sign-off)